sections/semantics-interactive-elements.include

```diff
@@ -42,10 +42,10 @@
     The <{details}> element is not appropriate for footnotes. Please see [[#footnotes]] for details on how to mark up footnotes.
   </p>
 
-  If the <{details}> element has a <{summary}> child element, then the first summary child element
-  <a>represents</a> the summary or legend of the details element. If there is no summary child
-  element, a user agent should provide its own legend (e.g. in English "Details"
-  or Spanish <span lang="es">"Detalles"</span>).
+  The first <{summary}> element child of the element, if any,
+  <a>represents</a> the summary or legend of the details. If there is no
+  child <{summary}> element, the user agent should provide its own legend (e.g.,
+  in English "Details" or Spanish <span lang="es">"Detalles"</span>).
```

Comment:
I could have been clearer in my review. By saying I didn't disagree, I meant that I thought your change is OK - but I wanted you to explain in more detail (as you did - thank you) why you made it, for my own benefit.

Probably most people who read this spec - and at least one editor - are not native speakers of english, so clear phrasing that people understand is important.

I won't mark this as request changes, but I would prefer to take your initial proposed change.